sup3r/sup3r.py

```diff
@@ -94,3 +98,69 @@ def _init_loggers(self, loggers=None, log_file='sup3r.log',
 
             logger.info('Active python environment versions: \n{}'
                         .format(pprint.pformat(VERSION_RECORD, indent=2)))
+
+    @classmethod
+    def init_pass_collect(cls, out_dir, file_paths, model_path, **kwargs):
```

Comment:
You should define what these kwargs go to, otherwise the API is difficult to read. You might also want to break this up into foward_pass_kwargs and collect_kwargs and define where the user can find a docstring of these options (consider linking to the API docs). 

Honestly I'm curious if we even need this high level SUP3R class... All it does is make directories and sets up configs? And honestly the user has to do all the work of setting up the configs anyway and just feeding them into the kwargs field so im not convinced this saves a ton of time.

Reply:
For the nsrdb, the high level NSRDB class is a common entry point for basically the entire software. This is more of a project setup utility... I'd suggest we name it as such and put it in the utilities or config sub modules.

Reply:
Yeah thats what I had in mind when I copied this SUP3R class in here. Should we just remove it after moving this method?

Reply:
Yeah I dont see a need to have a collection of entry points for all the sub-modules. For the NSRDB it was really to link the pipeline/cli to the pure python class methods